tests/pipeline/test_node.py

```diff
@@ -337,15 +337,15 @@ def dummy_func_args(**kwargs):
     [
         (
             inconsistent_input_size,
-            r"Inputs of function expected \[\'input1\'\], but got \[\'A\', \'B\'\]",
+            r"Inputs of function 'identity' expected \[\'input1\'\], but got \[\'A\', \'B\'\]",
```

Comment:
This looks nicer. Can we add some tests for `lambda` and `partial` when they are passed as the callable to the `node`? Not sure if it'll be as pretty.